_episodes/07-docker-image-examples.md

```diff
@@ -28,13 +28,36 @@ In this [GitHub Actions example](../_extras/e01-github-actions), you can learn m
 continuous integration in the clound and how you can use container images with GitHub to
 automate some of your workflow.
 
-<!--- Placeholder for 
+<!--- Placeholder for
 ## Geospatial Example
 
 Ask @mkuzak to make a PR to add extra for <https://github.com/escience-academy/docker-gdal-demo>
 
 -->
 
+## On an HPC Cluster
+
+It is possible to run containers on shared computing systems run by a university or national
+computing center. As a researcher, you can build and test your container on your own
+computer and then use it to run your full-scale computing work on a shared computing
+system like a high performance cluster or high throughput grid.
+
+The catch? Most university and national computing centers do not support *running*
+containers with Docker commands, and instead use a similar tool called Singularity or
+Shifter. However, both of these programs can be used to run Docker container images,
+so often people create their container as a Docker container image, so they can
+run it using Docker or Singularity.
+
+There isn't yet a working example of how to use Docker container images on a shared
+computing system, partially because each system is slightly different, but the
+following resources show what it can look like:
+
+- Singularity lesson
+- HPC Carpentry
+- Pawsey?
```

Comment:
```suggestion
- [Singularity lesson](https://carpentries-incubator.github.io/singularity-introduction/)
- [HPC Carpentry](https://www.hpc-carpentry.org/)
- [Pawsey](https://pawseysc.github.io/singularity-containers/)
```

Reply:
I had already made this change and forgotten to push. 🤦‍♀️